react/src/lib/List/index.js

```diff
@@ -52,6 +52,9 @@ class List extends React.Component {
 
   componentDidUpdate(prevProps, prevState) {
     const { listContext } = this.state;
+    if (this.props.shouldFocusActive && (prevProps.focus !== this.props.focus || prevProps.active !== this.props.active)) {
```

Comment:
Can we destructure the this.props props to shorten this up a bit? Other than that looks great.